src/utils/resources/template/hooks/useVmTemplateSource/utils.ts

```diff
@@ -99,11 +99,14 @@ export const getPVC = (name: string, ns: string) =>
     ns,
   });
 
-export const getDataSourcePVC = (name: string, ns: string) =>
+export const getDataSource = (name: string, ns: string) =>
   k8sGet({
     model: DataSourceModel,
     name,
     ns,
-  })
+  });
+
+export const getDataSourcePVC = (name: string, ns: string) =>
+  getDataSource(name, ns)
```

Comment:
@yaacov we are missing `V1BetaDataSource` in the api.
we need to cast `any` right now